### PR TITLE
Subwallet webhook demonstration [Alice-Faber Demo]

### PR DESCRIPTION
**Pull request by shaangill025**
Webhook dispatch is demonstrated inside the terminal as follows:
image

### CODE REVIEW

**Comment by ianco** [comment]
Web hooks are displayed optionally by providing an `--events` parameter, prob best to stick with this (i.e. display the web hok info optionally).

File: demo/runners/support/agent.py [hunk 1]
```diff
@@ -510,6 +524,11 @@ async def handle_webhook(self, topic: str, payload, headers: dict):
             wallet_id = headers.get("x-wallet-id")
             method = getattr(self, handler, None)
             if method:
+                self.log(
+                    f"Agent called controller webhook: POST {self.webhook_url}",
+                    (f"\nFor wallet: {wallet_id}" if wallet_id else ""),
+                    (f"\nPayload: {repr_json(payload)}" if payload else repr_json({})),
+                )
                 EVENT_LOGGER.debug(
```
Comment (ianco) [inline demo/runners/support/agent.py:532]: Web hook will be logged twice if run with `--events`

**Review by ianco — CHANGES_REQUESTED** [review]
Overall looks good!

File: demo/runners/support/agent.py (hunk 1 above)
Comment (shaangill025) [reply demo/runners/support/agent.py:527]: Now only relying on event_logger for demonstration.
image

**Review by ianco — APPROVED** [review]

**Review by andrewwhitehead — APPROVED** [review]

**Pull request merged**